templates/heroTab.hbs

```diff
@@ -0,0 +1,9 @@
+<div class="hero__sub-container">
+  <h2 class="hero__title heading-2">{{title}}</h2>
```

Comment:
We will need to have at least one h1 in the page. It would be possible to add somehow an if to make the first slide with an h1

Reply:
This was flagged and is still pending to be define by design. At the moment we will keep h2 as per the current designs.

Reply:
We can tackle that in a different ticket